src/Symfony/Component/Console/Application.php

```diff
@@ -62,6 +62,7 @@ class Application
     private $name;
     private $version;
     private $catchExceptions = true;
+    private $catchThrowables = false;
```

Comment:
What will you suggest instead?
We still have to differentiate exceptions from other throwables, as the last ones are currently handled in the fullstack by an exception handler, whereas exceptions are handled by the Console component itself.

We could make `catchExceptions` accept bit flags (`Application::CATCH_EXCEPTIONS`, `Application::CATCH_THROWABLES`) and consider `true` the same as `Application::CATCH_EXCEPTIONS` only. But I don't like it semantically and would complicate things.